services/movie/src/instrumentation.ts

```diff
@@ -13,9 +13,15 @@ import {
 } from '@opentelemetry/core';
 import { B3InjectEncoding, B3Propagator } from '@opentelemetry/propagator-b3';
 import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import {
+  ConsoleSpanExporter,
+  SimpleSpanProcessor,
+  WebTracerProvider,
+} from '@opentelemetry/sdk-trace-web';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
 
 const metricReader = new PrometheusExporter({
-  port: 8081,
+  port: 8082,
```

Comment:
Any reason for this change? having the same port while using different DNS in docker compose should not be a problem, unless you are trying to run both apps locally without docker compose